test/evd_test.cpp

```diff
@@ -0,0 +1,44 @@
+#include "gtest/gtest.h"
```

Comment:
```evd_test.cpp``` file needs to be added to tests/CMakeLists.txt together with other test files. Only then it will get compiled.

Reply:
Also the eigenvalues are wrong I think,  I get this in wolfram alpha  
λ_1≈12.7199
λ_2≈5.78305
λ_3≈-5.60024
λ_4≈2.09733

Reply:
Added the entry to the CMakelists.txt. Thanks for pointing it out.

Reply:
Regarding the low precision of the computed values, I think that it's probably due to the slightly different expressions used to compute the rotation matrix, which accumulates and propagates the floating point precision error over the iterations. I'll try to rewrite the expressions with lower number of flops in the subsequent refinements of the program.